src/IconToggle/index.js

```diff
@@ -2,13 +2,36 @@ import React, { Component } from 'react'
 import { View, StyleSheet } from 'react-native'
 import { IconToggle } from '@protonapp/react-native-material-ui'
 
+
+
 export default class WrappedIconToggle extends Component {
   handlePress = async () => {
     let {
       clickActions,
       input: { value, onChange },
     } = this.props
-    await onChange(!value)
+    //await onChange(!value)
+
+    let {
+      activeActions,
```

Comment:
You don't need to read these in separate from line 10. Just read in all of the actions at once.